Vagrantfile

```diff
@@ -75,16 +86,16 @@ fi
 # mv /tmp/consul /usr/bin) || exit 1
 
 # add vagrant user to docker group
-(usermod -a -G docker vagrant)
+# (usermod -a -G docker vagrant)
```

Comment:
I think @shaleman added this for ease of running docker commands as vagrant user without having to sudo. Don't we need it anymore?